src/core/FxMixer.cpp

```diff
@@ -172,8 +172,9 @@ void FxChannel::doProcessing()
 		m_stillRunning = m_fxChain.processAudioBuffer( m_buffer, fpp, m_hasInput );
 
 		Mixer::StereoSample peakSamples = Engine::mixer()->getPeakValues(m_buffer, fpp);
-		m_peakLeft = qMax( m_peakLeft, peakSamples.left * v );
-		m_peakRight = qMax( m_peakRight, peakSamples.right * v );
+		
+		m_peakLeft = peakSamples.left * v;
+		m_peakRight = peakSamples.right * v;
```

Comment:
Source for potential bugs?

Reply:
For reference: there used to be a QMax to ensure it doesn't cross some maximum amount (peakSamples.left or .right respectively I believe)